src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/DeviceTest.cs

```diff
@@ -29,7 +29,8 @@ protected override void CleanupTest ()
 				string deviceLog = Path.Combine (outputDir, "logcat-failed.log");
 				string remote = "/data/local/tmp/screenshot.png";
 				RunAdbCommand ($"shell screencap {remote}");
-				RunAdbCommand ($"logcat -d > \"{deviceLog}\"");
+				var output = RunAdbCommand ($"logcat -d");
```

Comment:
We should use `$"logcat -d -f \"{deviceLog}\""`:

```bash
$ adb logcat --help
Usage: logcat [options] [filterspecs]
options include:
  -s              Set default filter to silent. Equivalent to filterspec '*:S'
  -f <file>, --file=<file>               Log to file. Default is stdout
…
```

Reply:
That would not work as `-f` option writes the file to the device and not to the host. Have been there :-)